tests/integration/test_services.py

```diff
@@ -0,0 +1,66 @@
+# package imports
+import logging
+import os
+import unittest
+
+import earthaccess
+from earthaccess.api import search_datasets
+from vcr.unittest import VCRTestCase  # type: ignore[import-untyped]
+
+assertions = unittest.TestCase("__init__")
+logger = logging.getLogger(__name__)
+
+assertions.assertTrue("EARTHDATA_USERNAME" in os.environ)
+assertions.assertTrue("EARTHDATA_PASSWORD" in os.environ)
+
+logger.info(f"Current username: {os.environ['EARTHDATA_USERNAME']}")
+logger.info(f"earthaccess version: {earthaccess.__version__}")
+
+
+class TestServices(VCRTestCase):
+    def scrub_access_token(self, string, replacement=""):
+        def before_record_response(response):
+            body_string = str(response["body"]["string"])
+            if "access_token" in body_string:
+                response["body"]["string"] = "ACCESS_TOKEN"
+            if "uid" in body_string:
+                response["body"]["string"] = "UID"
+            return response
+
+        return before_record_response
+
```

Comment:
@nikki-t, if you add the following method, I think this should fix the failing unit tests:

```python
    def _get_vcr_kwargs(self, **kwargs):
        kwargs["decode_compressed_response"] = True
        return kwargs
```

Reply:
@nikki-t Do you think you can manage to try this this week? It would be nice to get this PR landed. You'll have to merge changes from `main` because it looks like you have conflicts in the changelog.

Reply:
@chuckwondo - I won't have time this week since it's a short week but I should have a little time next week to try to merge the changes in main and implement the fix you mentioned.